integr8ly/integreatly-operator
# [INTLY-5214] Check all alerts as part of e2e & functional tests

### Description 
Check all alerts exist as part of e2e and functional tests 
- Alerts we expected that weren't found 
- Alerts that were found and not expected 

### Verification
- Log in to an existing Openshift 4 cluster 
- Install integreatly from master
- Check out this branch and run `make test/functional`. The test is common to both e2e and functional tests, so running the functional ones is just quicker 
- See that the output of the test is as expected (e.g. try comment out some of the expected alerts, verify that the report output is correct)
